base/random.jl

```diff
@@ -1384,7 +1406,7 @@ function randexp_unlikely(rng, idx, x)
 end
 
 """
-    randn!([rng=GLOBAL_RNG], A::AbstractArray) -> A
+    randn!([rng::AbstractRNG=GLOBAL_RNG,] A::AbstractArray) -> A
```

Comment:
Have we a convention for comma position relative to brackets in docstring signatures? If so, what is it? :) (Switched from commas outside the brackets to inside the brackets here and likewise for `randexp!` below.)